pkg/storaged/content-views.jsx

```diff
@@ -86,13 +86,23 @@ export function set_crypto_options(block, readonly, auto) {
             if (!auto)
                 opts.push("noauto");
         }
+        if (nofail !== null) {
+            extract_option(opts, "nofail");
+            if (nofail)
+                opts.push("nofail");
+        }
+        if (netdev !== null) {
+            extract_option(opts, "_netdev");
+            if (netdev)
+                opts.push("_netdev");
+        }
         config.options = { t: 'ay', v: utils.encode_filename(unparse_options(opts)) };
         return commit();
     });
 }
 
 export function set_crypto_auto_option(block, flag) {
-    return set_crypto_options(block, null, flag);
+    return set_crypto_options(block, null, flag, null, null);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221115-130151-a219e579-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)